server/controllers/finance/reports/invoices/receipt.pos.handlebars

```diff
@@ -36,6 +36,7 @@
 </table>
 {{> underline}}
 <h2 style="text-align : right;">{{translate 'FORM.LABELS.TOTAL'}}: <b>{{currency cost metadata.enterprise.currency_id}}</b></h2>
+<h4 style="text-align : right;"> <b>{{numberToText cost lang metadata.enterprise.currencyName}}</b></h>
```

Comment:
Just like `translate` doesn't require the language, it would be nice to make this filter not require the language.  I believe that code is found in `html.js`, but I'm not certain.